content/es/docs/concepts/architecture/control-plane-node-communication.md

```diff
@@ -0,0 +1,100 @@
+---
+reviewers:
+- dchen1107
+- liggitt
+title: Communication between Nodes and the Control Plane
+content_type: concept
+weight: 20
+aliases:
+- master-node-communication
+---
+
+<!-- overview -->
+Este documento cataloga las rutas de comunicación entre el servidor {{< glossary_tooltip term_id="kube-apiserver" text="servidor API" >}} y el {{< glossary_tooltip text="cluster" term_id="cluster" length="all" >}} de Kubernetes. El objetivo es permitir a los usuarios personalizar su instalación para reforzar la configuración de la red de tal manera que el clúster pueda ejecutarse en una red no confiable (o en IPs públicas completas en un proveedor de servicios en la nube).
+
+<!-- body -->
+
+## Nodo a Control Plane
```

Comment:
We have used "plano de control" as the translation for "control plane" in other Spanish articles. For instance, the official concept meaning translation:

https://kubernetes.io/es/docs/concepts/overview/components/#componentes-del-plano-de-control

Please use "Plano de Control" to keep consistency.

```suggestion
## Nodo a Plano de Control
```